### PR TITLE
Add Speed Run Ethereum to develop tooling

### PR DESCRIPTION
**Pull request by wackerow**
- Adds the Speed Run Ethereum site to the list of developer tools
- Placed under "bootcamp" to start... curious people thoughts on this. Could potentially also lit as a "sandbox"

Fixes https://github.com/ethereum/ethereum-org-website/issues/7562

### CODE REVIEW

**Comment by corwintines** [comment]
I think this could be in both bootcamps and sandbox personally

**Review by corwintines — APPROVED** [review]
I think this could be in both sandbox and bootcamp, but dont see that as blocking

File: src/intl/en/page-developers-learning-tools.json [hunk 1]
```diff
@@ -35,6 +35,8 @@
   "page-learning-tools-remix-logo-alt": "Remix logo",
   "page-learning-tools-sandbox": "Code sandboxes",
   "page-learning-tools-sandbox-desc": "These sandboxes will give you a space to experiment with writing smart contracts and understanding Ethereum.",
+  "page-learning-tools-speed-run-ethereum-description": "Use Scaffold-ETH to expedite your Solidity knowledge. When you are ready to test your knowledge, speed run Ethereum.",
```
Comment (minimalsm) [inline src/intl/en/page-developers-learning-tools.json:38]: This doesn't really explain what speed-run Ethereum is.

**Review by minimalsm — COMMENTED** [review]
Yeah, we definitely need to rework this page's sections (too much in bootcamps) but I'm good with putting it here for now.

Should we tweak the description a little? It isn't immediately obvious what speed run Ethereum is from its description.

File: src/intl/en/page-developers-learning-tools.json (hunk 1 above)
Comment (corwintines) [inline src/intl/en/page-developers-learning-tools.json:38]:
```suggestion
  "page-learning-tools-speed-run-ethereum-description": "SpeedRunEthereum is a set of challenges to start developing on Ethereum uring Scaffold-ETH",
```
Maybe something like this?

Comment (wackerow) [reply src/intl/en/page-developers-learning-tools.json]: yeah that sounds good, I just ripped some text from their website =)

**Pull request merged**